README.md

```diff
@@ -51,8 +51,6 @@
 
 Gets your timetable from ERP and adds it to your Google Calendar or gives you an ICS file which you can add in any common calendar application.
 
-> **Note** All updates to this repo should reflect, with appropriate refactorisation, in [gyft-serve](https://github.com/metakgp/gyft-serve/)
```

Comment:
I think we should keep this PR focused on the relevant changes. It would be great if you open a separate PR updating the documentation since it is a bit out of date.

Reply:
reverted back the change

Reply:
Can you rebase and make the commit vanish?